pages/en/new-design/404.mdx

```diff
@@ -0,0 +1,16 @@
+---
+layout: centered.hbs
```

Comment:
I just discovered that we ignore this little fella even on static exports. Sooo, to be honest, we can just get rid of this file. Since on static exports Next.js also generates a default 404.html file based on our `not-found.tsx` then let's do the following:

- Delete this file (404.mdx)
- Let's actually use i18n strings
- Let's move this content to the `not-found.tsx` so we don't need to use the dynamicRouter
- And then you can remove this https://github.com/nodejs/nodejs.org/blob/main/next.dynamic.constants.mjs#L19